saleor/checkout/utils.py

```diff
@@ -881,3 +885,22 @@ def create_order(cart, tracking_code, discounts, taxes):
 
     _fill_order_with_cart_data(order, cart, discounts, taxes)
     return order
+
+def is_fully_paid(cart: Cart):
```

Comment:
Module-level function definitions not separated by two blank lines? Or are you leaving this to Yapf?